### PR TITLE
Shell scripts for match command on K8

### PR DESCRIPTION
**Pull request by norabelrose**
Now working

### CODE REVIEW

**Comment by norabelrose** [comment]
Oh also I need to remove the submodule changes I guess? Idk why that's showing up

File: kubernetes/launch-match.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# shellcheck disable=SC2046
+export $(grep -v '^#' "$GIT_ROOT"/kubernetes/active-images.env | xargs)
+
+ADVERSARY_PATH="$3"
+VICTIM_PATH=/shared/victims/"$2"
+
+# shellcheck disable=SC2215,SC2086,SC2089,SC2090
+ctl job run --container \
+    "$CPP_IMAGE" \
+    --volume-name go-attack \
```
Comment (AdamGleave) [inline kubernetes/launch-match.sh:72]: There's a regression in functionality here relative to https://github.com/HumanCompatibleAI/go_attack/blob/main/kubernetes/launch-job.sh#L74 that supports both Weka and non-Weka. Note we can't do Weka from the CHAI nodes.

**Review by AdamGleave — COMMENTED** [review]
Basic approach LGTM.

There's a fair amount of code duplication between this and https://github.com/HumanCompatibleAI/go_attack/blob/main/kubernetes/launch-job.sh#L74 -- I'd suggest trying to factor some of these out into a common part?

In particular, the Git porcelain check is in both, and could be turned into a function `check_git_clean()` or just something that's always run when you source the common script.

The Kubernetes updating images and sourcing the active images part is also shared.

The Weka vs CHAI thing isn't present here but you probably want it -- so that's something you could also share.

The CLI args parsing is a bit duplicated but I think on balance probably best to leave it as is -- it's pretty simple code, and there's enough differences that it'd be a pain to combine them.

**Comment by tomtseng** [comment]
I didn't see this and wrote the same PR at #48. I'll close #48.

Some things you may want to copy from #48:
* Currently, all replicas are going to write into the same file `-log-file /shared/match/"$RUN_NAME"/match.log`. I think they all append to the file but it may be less confusing to have replicas write into different files
* I have some code for `update_images.py` to allow only building the `cpp` image, since building the `python` image is unnecessary for `match`

**Comment by AdamGleave** [comment]
Sorry for the duplication Tom! Those look like some cool features -- Nora you should definitely cherrypick some of those.

Please request a re-review from me once those changes integrated + code duplication reduced.

**Comment by norabelrose** [comment]
I decided that #48 was actually a better PR to work off of anyway, so I re-opened it and made some commits. Closing this one now.

**Pull request closed without merging**